op-acceptance-tests/tests/custom_gas_token/helpers.go

```diff
@@ -0,0 +1,70 @@
+// op-acceptance-tests/tests/custom_gas_token/helpers.go
+package custom_gas_token
+
+import (
+	"context"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/lmittmann/w3"
+)
+
+var (
+	// L2 predeploy: L1Block (address is stable across OP Stack chains)
+	l1BlockAddr = common.HexToAddress("0x4200000000000000000000000000000000000015")
+
+	// L2 predeploy: L2CrossDomainMessenger & L2StandardBridge (for revert checks)
+	l2XDMAddr    = common.HexToAddress("0x4200000000000000000000000000000000000007")
+	l2BridgeAddr = common.HexToAddress("0x4200000000000000000000000000000000000010")
```

Comment:
I think this address already define in https://github.com/okx/optimism/blob/2ce4256e0c0c2141f91e7f7b394a3d36e27218fd/devnet-sdk/contracts/constants/constants.go#L8-L46. May be use these?